Apps/W1/ShopifyConnector/app/Customer/Codeunits/ShpfyCreateCustomer.Codeunit.al

```diff
@@ -0,0 +1,136 @@
+/// <summary>
+/// Codeunit Shpfy Create Customer (ID 30110).
+/// </summary>
+codeunit 30110 "Shpfy Create Customer"
+{
+    Access = Internal;
+
+    TableNo = "Shpfy Customer Address";
+
```

Comment:
This is correct, because the permissions are already in the role of "D365 BASIC"
Previous we use the permission sets in an xml file, because the permission objects didn't exists.
With the XML file, we must also included these permissions, but now with the permission set objects is this not necessary anymore.

Reply:
Updated with commit: "Inderect Permissions"